examples/tts/tts_infer.py

```diff
@@ -137,7 +137,7 @@ def create_infer_dags(
         labels=tacotron2_params['labels'],
         batch_size=infer_batch_size,
         num_workers=cpu_per_dl,
-        load_audio=False,
+        # load_audio=False,
```

Comment:
please revert this change.  it helps clarity to keep it explicitly even if load_audio is False by default.